tests/unit/test.program.js

```diff
@@ -73,8 +73,8 @@ describe('program.Program', () => {
       shouldExitProgram: true,
     })
       .then(() => {
-        assert.equal(fakeProcess.exit.called, true);
-        assert.equal(fakeProcess.exit.firstCall.args[0], 1);
+        sinon.assert.calledOnce(fakeProcess.exit);
+        sinon.assert.calledWithMatch(fakeProcess.exit, 1);
```

Comment:
I see, my confusion was thinking that (==) was strict equality. Thank you for clearing this up.